lld/test/MachO/implicit-and-allowable-clients.test

```diff
@@ -0,0 +1,49 @@
+# REQUIRES: arm
```

Comment:
Shouldn't this be AArch64, not arm?

Reply:
I thought `arm` implied `aarch64`, doesn't it? I can change it. It is actually not important which architecture is used for building. I just needed one.

Reply:
Seems that it has already been changed in ce863dd5a77723fc21e1e43f5d3dd503e9a73d14.

I was under the impression that the arm backend implied both of them. I see there's two backends there. I will try not to forget for next time.

Reply:
It does not, arm and aarch64 are distinct, but I see @MaskRay has already updated the REQUIRES line in the test.